src/main/java/org/folio/circulation/domain/representations/ItemPickSlipRepresentation.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.circulation.domain.representations;
+
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+
+import io.vertx.core.json.JsonObject;
+
+import org.folio.circulation.domain.Item;
+import org.folio.circulation.domain.Location;
+
+public class ItemPickSlipRepresentation {
+
+  public JsonObject create(Item item) {
+    if(item == null || item.isNotFound()) {
+      return new JsonObject();
+    }
+
+    JsonObject itemSummary = new JsonObject();
+
+    write(itemSummary, "id", item.getItemId());
+    write(itemSummary, "holdingsRecordId", item.getHoldingsRecordId());
+    write(itemSummary, "instanceId", item.getInstanceId());
+    write(itemSummary, "title", item.getTitle());
+    write(itemSummary, "barcode", item.getBarcode());
+    write(itemSummary, "callNumber", item.getCallNumber());
```

Comment:
The description of the `callNumber` in schema says `"The call number from the holdings record",` however, `item.getCallNumber()` is effective call number (either `itemLevelCallNumber` if present or `holdings.callNumber`).